src/Hedgehog/Property.fs

```diff
@@ -81,11 +81,32 @@ module Property =
     let bind (k : 'a -> Property<'b>) (m : Property<'a>) : Property<'b> =
         bindGen (toGen << k) (toGen m) |> ofGen
 
+#if !FABLE_COMPILER
```

Comment:
Can you use positive logic by writing the compiler directives this way?
```
#if FABLE_COMPILER
...
#else
...
#endif
```

Reply:
Done